index.js

```diff
@@ -4,11 +4,79 @@ const graphqlHTTP = require('express-graphql');
 const cors = require('cors');
 const root = require('./src/resolvers/RootResolver');
 const schema = require('./src/schema/schema');
+const GitHubStrategy = require('passport-github2').Strategy;
+const passport = require('passport');
+const partials = require('express-partials');
```

Comment:
@Gwillison415 I'm not convinced you need this. Please investigate and either comment that you need it, or, remove, thanks!